articles/relnotes/index.md

```diff
@@ -16,6 +16,18 @@ For installation instructions, please refer to the [install guide](xref:microsof
 
 For update instructions, please refer to the [update guide](xref:microsoft.quantum.update).
 
+## Version 0.XX.XXXX.XXXX
+
+*Release date: December 5th, 2019*
+
+This release contains the following:
+
+- New Test attribute for Q# unit testing, see [here](xref:https://docs.microsoft.com/en-us/quantum/techniques/testing-and-debugging?view=qsharp-preview&tabs=tabid-vs2019) for updated documentation
```

Comment:
Can we link to the API documentation for the new `@Test()` attribute?

Separately, I don't think the link will work as formatted, since `xref` denotes that a URI references a DocFX UID and not a conventional URL. Also, I'd suggest leaving out the query string and "en-us" from the docs link:
```suggestion
- New Test attribute for Q# unit testing, see [here](https://docs.microsoft.com/quantum/techniques/testing-and-debugging) for updated documentation
```

Reply:
do we have a link to the new API docs?